config/cable.yml

```diff
@@ -0,0 +1,9 @@
+development:
```

Comment:
I don't think we want this? I believe it's from ActionCable and it's wrong in production